src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestHelpers/ExclusiveRunner.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+
+namespace Microsoft.PowerFx.Core.Tests
+{
+    /// <summary>
+    /// Ensures only one instance can exist at a time.
+    /// </summary>
+    public abstract class ExclusiveRunner : IDisposable
```

Comment:
>ExclusiveRunner [](http://example.com/codeflow?start=26&length=15)

Yikes... ... this is painful...

I guess this is because FeatureFlags is static?

Reply:
even this isn't fully correct since we may run against other tests that call RecalcEngine directly (outside of BaseRunner).

Reply:
Is there something we can do at the Xunit level? ie: 
[https://xunit.net/docs/running-tests-in-parallel](https://xunit.net/docs/running-tests-in-parallel)

Reply:
Yes, this is due to static FeatureFlags
We should probably consider changing them to non-static, maybe in PowerFxConfig or elsewhere

Didn't know Xunit was providing "Collection" feature for tests, it's exactly what we need.